Nur in pvr350-2007-08-07: libvdr-pvr350.so.
Nur in pvr350-2007-08-07: libvdr-pvr350.so.1.6.0.
Nur in pvr350-2007-08-07: patches.
Nur in pvr350-2007-08-07: pvr350audio.o.
Nur in pvr350-2007-08-07: pvr350device.o.
Dateien pvr350-2007-08-07-orig/pvr350.o und pvr350-2007-08-07/pvr350.o sind verschieden.
diff -ru pvr350-2007-08-07-orig/pvr350osd.c pvr350-2007-08-07/pvr350osd.c
--- pvr350-2007-08-07-orig/pvr350osd.c	2007-07-28 22:43:32.000000000 +0200
+++ pvr350-2007-08-07/pvr350osd.c	2008-04-01 13:54:20.315786872 +0200
@@ -24,8 +24,8 @@
 #include "pvr350device.h"
 
 #if VDRVERSNUM >= 10307
-cPvr350Osd::cPvr350Osd(int Left, int Top,int fbfd, unsigned char *osdbuf)
-    :cOsd(Left, Top) 
+cPvr350Osd::cPvr350Osd(int Left, int Top, uint level, int fbfd, unsigned char *osdbuf)
+    :cOsd(Left, Top, level) 
 {
     fd = fbfd;
     osd = osdbuf; 
@@ -130,9 +130,9 @@
     osdfd  = fd;
 }
 
-cOsd *cPvr350OsdProvider::CreateOsd(int Left, int Top)
+cOsd *cPvr350OsdProvider::CreateOsd(int Left, int Top, uint level)
 {
-  return new cPvr350Osd(Left, Top, osdfd, osdBuf);
+  return new cPvr350Osd(Left, Top, level, osdfd, osdBuf);
 }
 
 #else
diff -ru pvr350-2007-08-07-orig/pvr350osd.h pvr350-2007-08-07/pvr350osd.h
--- pvr350-2007-08-07-orig/pvr350osd.h	2007-07-26 23:11:20.000000000 +0200
+++ pvr350-2007-08-07/pvr350osd.h	2008-04-01 13:40:03.000000000 +0200
@@ -20,7 +20,7 @@
     unsigned char *osd;
 
 public:
-    cPvr350Osd(int Left, int Top,int fbfd, unsigned char *osdbuf);
+    cPvr350Osd(int Left, int Top, uint level, int fbfd, unsigned char *osdbuf);
     ~cPvr350Osd();
     eOsdError CanHandleAreas(const tArea *Areas, int NumAreas);
     void Flush(void);
@@ -36,7 +36,7 @@
     int            osdfd; 
 public:
     cPvr350OsdProvider(int fd, unsigned char *buf);
-    virtual cOsd *CreateOsd(int Left, int Top);
+    virtual cOsd *CreateOsd(int Left, int Top, uint level);
 };
 
 #else
Nur in pvr350-2007-08-07: pvr350osd.o.
Nur in pvr350-2007-08-07: pvr350tools.o.
